src/main/webapp/react-layout/react-layout.js

```diff
@@ -4,8 +4,8 @@ import { makeStyles } from '@material-ui/core/styles'
 
 import GoldenLayout from 'golden-layout'
 
-import 'golden-layout/src/css/goldenlayout-base.css'
-import 'golden-layout/src/css/goldenlayout-light-theme.css'
+//import 'golden-layout/src/css/goldenlayout-base.css'
+//import 'golden-layout/src/css/goldenlayout-light-theme.css'
```

Comment:
Do conditional importing for GL on SSR

Reply:
I think if you condition higher up the view tree, you won't need to do it here.